.azure_pipelines/docker_images/ubuntu_1804_py_37_cuda_101_cudnn_7_dev/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# This is the Dockerfile for an image that is ready to build PyTorch from source.
```

Comment:
This script does not seem to be specific to Azure pipelines, is it?
If that the case, can we perhaps move it to `docker` folder?

Reply:
It is not specific to Azure pipelines. I will move the Dockerfiles to the `docker` folder.